options/locale/locale_en-US.ini

```diff
@@ -2576,6 +2577,7 @@ publish_release  = `released <a href="%s/releases/tag/%s"> "%[4]s" </a> at <a hr
 review_dismissed = `dismissed review from <b>%[4]s</b> for <a href="%[1]s/pulls/%[2]s">%[3]s#%[2]s</a>`
 review_dismissed_reason = Reason:
 create_branch = created branch <a href="%[1]s/src/branch/%[2]s">%[3]s</a> in <a href="%[1]s">%[4]s</a>
+find_file = Go to file
```

Comment:
Why do you name it `find_file` when do you mean `Go to file`?

Reply:
Learn from Github ...